cachecontrol/compat.py

```diff
@@ -30,3 +30,5 @@
     text_type = unicode
 except NameError:
     text_type = str
+
+__all__ = ["urljoin", "pickle", "HTTPResponse", "is_fp_closed", "text_type"]
```

Comment:
since setup.py requires recent python https://github.com/ionrock/cachecontrol/blob/6155c199438bfc3944a833f1dec6fb74134476f1/setup.py#L27

I suppose some or more of this file could be removed